parser/mongodatastore.go

```diff
@@ -117,6 +117,14 @@ func (mongo *MongoDatastore) Index() {
 			}
 		}
 		collMap.rwLock.Unlock()
+
+		err := mongo.metaDB.MarkDBImported(database, true)
+		if err != nil {
+			mongo.logger.WithFields(log.Fields{
```

Comment:
It looks like this `err` will get logged twice, once inside of the `MarkDBImported` function and once here. Is there a specific reason for that?  I think this is just me not understanding the logging mechanism very well so I thought I'd ask in order to learn something.

Reply:
Nah, its exactly the way you explain it. It gets double logged. Good catch